front/lib/api/assistant/messages.ts

```diff
@@ -563,7 +563,7 @@ export async function fetchMessageInConversation(
   conversation: ConversationWithoutContentType,
   messageId: string
 ) {
-  return Message.findOne({
+  const message = await Message.findOne({
```

Comment:
😮

Reply:
What is the goal of the change ?

Reply:
none since i removed the AgentStepContent include
removing